src/components/ProgressGuide/FollowDialog.tsx

```diff
@@ -762,7 +762,7 @@ function SearchInput({
         autoComplete="off"
         autoCapitalize="none"
         accessibilityLabel={_(msg`Search profiles`)}
-        accessibilityHint={_(msg`Search profiles`)}
+        accessibilityHint={_(msg`Search for profiles`)}
```

Comment:
Consider "Searches for profiles" "Search for" is imperative.

Reply:
@jason-me In case it's helpful to know, when reviewing a PR you can click the button with the symbol of a page with a ± and this lets you provide a suggestion that can then be easily accepted by the author :)

E.g.:
```suggestion
        accessibilityHint={_(msg`suggested text here`)}
```

Reply:
Thanks for the tip!